cylc/flow/workflow_files.py

```diff
@@ -340,9 +340,7 @@ class RemoteCleanQueueTuple(NamedTuple):
 """
 
 SUITERC_DEPR_MSG = (
-    f"Backward compatibility mode ON for CYLC 7 '{WorkflowFiles.SUITE_RC}'"
-    " files: please address deprecation warnings and upgrade to Cylc 8 graph"
-    f" syntax BEFORE renaming the file to '{WorkflowFiles.FLOW_FILE}'.\n"
+    f"Backward compatibility mode ON for CYLC 7 '{WorkflowFiles.SUITE_RC}'."
```

Comment:
```suggestion
    f"Backward compatibility mode ON for CYLC 7 '{WorkflowFiles.SUITE_RC}' config files. "
    f"When ready to upgrade, rename the file to {WorkflowFiles.FLOW_FILE} then address "
    "any resulting validation errors and warnings."
```

Reply:
Maybe "any resulting validation ~errors and~ warnings" because there shouldn't be any errors, only warnings?

Reply:
> because there shouldn't be any errors

I think errors are quite likely with optional outputs e.g. `a => b; a:fail > c`.

Good enough?:

```suggestion
    f"Backward compatibility mode ON for Cylc 7 '{WorkflowFiles.SUITE_RC}'."
    f"\nTo upgrade to Cylc 8, rename {WorkflowFiles.SUITE_RC}"
    f" to {WorkflowFiles.FLOW_FILE}."
```

Reply:
Ah ok. I would go for Hilary's suggestion, as I think it's useful to give notice that the user will get warnings/errors